programs/ssl/ssl_server2.c

```diff
@@ -406,6 +423,10 @@ struct options
     const char *key_file;       /* the file with the server key             */
     const char *crt_file2;      /* the file with the 2nd server certificate */
     const char *key_file2;      /* the file with the 2nd server key         */
+    const char *async_operations; /* supported SSL asynchronous operations  */
+    int async_private_delay1;   /* number of times f_async_resume needs to be called for key 1, or -1 for no async */
+    int async_private_delay2;   /* number of times f_async_resume needs to be called for key 2, or -1 for no async */
```

Comment:
Because `ssl_server2` only supports two keys. And even if it supported more, the only reason the async testing code supports two keys is to test the fallback mechanism (callback returning `MBEDTLS_ERR_SSL_HW_ACCEL_FALLTHROUGH`).

We aren't very good at documenting our SSL test strategy. If we did, this wouldn't be the place, it would be either a comment in `ssl-opt.sh` (which there is to explain the test, but not to explain why no other test is needed on that count) or ideally a separate test plan document.

Reply:
OK, so it is existing that `ssl_server2` only supports two keys. This is fine.